addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -529,8 +529,8 @@
 		
 		"154"	//Pain Train
 		{
-			"desp"			"Pain Train: {positive}+30% health from healers, {negative}-50% health from health packs"
-			"attrib"		"70 ; 1.3 ; 109 ; 0.5 ; 68 ; 0.0 ; 67 ; 1.0"
+			"desp"			"Pain Train: {negative}+10% damage vulnerability from all sources"
```

Comment:
Does the ``from all sources`` really need to be specified explicitly? Attribute description itself is ``%s1% damage vulnerability on wearer``, should be clear enough imo

Reply:
I felt like it needed to be specified because the weapon already has some kind of damage vulnerability, so telling the player it moves from x to y type of vulnerability should be more important than leaving it at just "damage," possibly making someone wonder "doesn't it already have that stat"

Reply:
`from all sources` does sound better when weapon already have bullet vulnerability